### PR TITLE
Create abstraction for braking via relay pin

### PR DESCRIPTION
**Pull request by ryescholin**

### CODE REVIEW

File: pod-operation/src/components/brakes.rs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+use tracing::debug;
+
+use rppal::gpio::{Gpio, OutputPin};
+
+pub struct Brakes {
+	pin: OutputPin,
+}
+
+const PIN_BRAKES: u8 = 5; //Replace with real value
```
Comment (taesungh) [inline pod-operation/src/components/brakes.rs:9]: Chore: the actual relay pin number we're using should be 26 based on last year

File: pod-operation/src/components/brakes.rs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+	pub fn disable(&mut self) {
+		debug!("Disabling brakes.");
+		self.pin.set_low();
+	}
+
+	pub fn enable(&mut self) {
+		debug!("Enabling brakes.");
+		self.pin.set_high();
+	}
```
Comment (taesungh) [inline pod-operation/src/components/brakes.rs:26]: My apologies for not clarifying in the original issue description, but the brakes operate on an inverted actuation.
Could we rename these two methods to `engage` and `disengage` where setting the pin high disengages and setting the pin low engages?

File: pod-operation/src/components/brakes.rs (hunk 1 above)
Comment (ryescholin) [reply pod-operation/src/components/brakes.rs:9]: perf

File: pod-operation/src/components/brakes.rs (hunk 2 above)
Comment (ryescholin) [reply pod-operation/src/components/brakes.rs:26]: yes ofc

**Review by taesungh — APPROVED** [review]
Simple enough, thanks for working on this.

**Review by samderanova — APPROVED** [review]

**Pull request merged**